Dockerfile.template

```diff
@@ -170,4 +170,11 @@ ENTRYPOINT ["docker-entrypoint.sh"]
 {{ ) else ( -}}
 USER haproxy
 {{ ) end -}}
+
+{{ if [ "2.0", "2.2", "2.4", "2.5", "2.6" ] | index(env.version) then ( -}}
+# no WORKDIR for backwards compatibility (to try to avoid breaking existing users)
+{{ ) else ( -}}
+WORKDIR /var/lib/haproxy
```

Comment:
Maybe we should include a comment here, so we remember why we set this and don't remove it later? :sweat_smile:

```suggestion
# https://github.com/docker-library/haproxy/issues/200
WORKDIR /var/lib/haproxy
```

(Otherwise this is great, thank you! :bow:)

Reply:
Okay, done.